packages/bson-transpilers/test/yaml/non-idiomatic.yaml

```diff
@@ -1,209 +1,181 @@
-runner: !!js/function >
-    (it, type, expect, input, output, transpiler, test) => {
-        it(test.description, () => {
-            expect(
-                transpiler[input][output].compile(test.input[input], false)
-            ).to.equal(test.output[output]);
-        });
-    }
+runner: !<tag:yaml.org,2002:js/function> |-
```

Comment:
I'm sorry, but what is this? Is this to have syntax highlighting?

Reply:
Ah yeah, i don't know :). I did most of the changes with a script that parsed the yaml files, changes / replaces the tests, and wrote the file back with Yaml.dump. The library generated that, i'm assuming is a fancier way to write this: `!!js/function`